htdocs/editor_tab.js

```diff
@@ -413,6 +413,14 @@ var editor = function () {
             rcloud.stars.get_my_starred_notebooks()
         ])
             .spread(function(user_notebook_set, my_stars_array) {
+
+                rcloud.get_users().
+                then(function(data){
+                    window.allTheUsers = data;
+                });
+
```

Comment:
let's figure out how to only call `get_users` once, since it's expensive. Means refactoring the `.all()` above.